wiki/multilang_md.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+import re
+import click
+import emoji
+from collections import OrderedDict  # toc
+
+
+def remove_emoji(text):
+    emoji_pattern = re.compile(
+        "["
+        "\U0001F600-\U0001F64F"  # emoticons
+        "\U0001F300-\U0001F5FF"  # symbols & pictographs
+        "\U0001F680-\U0001F6FF"  # transport & map symbols
+        "\U0001F1E0-\U0001F1FF"  # flags (iOS)
+        "]+",
+        flags=re.UNICODE,
+    )
+    return emoji_pattern.sub(r"", text)  # no emoji
+
+
+def create_toc(_dict, min_, max_, enable_emoji_header, _toc_str="", _level=1):
+    for head, child in _dict.items():
+        if (min_ <= _level) and (_level <= max_):
+            # head
+            if not enable_emoji_header:
+                head = remove_emoji(head)
+            # suburl
+            spc = r"[`~!@#$%\^&\*\(\)_=\+\|\[\]\{\}\\\\;:'\",./<>\?]+"
+            suburl = re.sub(spc, "", head)
+            suburl = remove_emoji(suburl)
+            suburl = suburl.replace("  ", " ").replace(" ", "-")
+            # toc line
+            temp = f"1. [{head}](#{suburl})\n"
+            # append
+            _toc_str += "    " * (_level - min_)
+            _toc_str += temp
+        # recursive
+        _toc_str = create_toc(
+            child, min_, max_, enable_emoji_header, _toc_str, _level + 1
+        )
+    return _toc_str
+
+
+class CreateMonolangualDoc(object):
+    def __init__(self, path, file_name, lang, suffix=True):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/mommel/hs-lr-midi-schwurbler/pullRequest?prid=6419091)